src/Cms/UserRules.php

```diff
@@ -299,6 +299,10 @@ public static function validEmail(User $user, string $email, bool $strict = fals
      */
     public static function validId(User $user, string $id): bool
```

Comment:
`UserRules::validId` should be used in `UserActions::createId()` instead of only `$this->kirby()->users()->has($id)`.

Reply:
Absolutely! But how do you write a try/catch in a while clause condition?

Reply:
Something like that? 

```php
while(true) {
    try {
        UserRules::validId($id);
        break;
    } catch (Throwable e) {
        continue;
    }
}
```

Reply:
Not the prettiest, but could do the job.

Reply:
Ah, I found it.